account_fiscal_position_rule_sale_stock/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends the sale stock.
```

Comment:
I suggest instead:

This module enables the account_fiscal_position_rule features (see module README) in the context of a sale order with a delivery order (sale and stock modules).